WordPress/gutenberg
# Remove the site editor sidebar navigation edit button

## What?


The site editor navigation section lists all available navigations. Focusing on one exposes an edit button which opens the focused navigation edit mode. Let's remove this button.

## Why?


Editing navigation in isolation never amounted to much and that button creates the illusion that the navigation is not editable in place already. Since we have some discoverability issues around editing menus using the navigation block we don't want any more confusion on top.

## How?


Just removed the button.

## Testing Instructions





1. Make sure to have some navigations created
2. Go to the site editor > navigation
3. Select a navigation
4. **Notice in the up right side of the sidebar there is no edit button**

### Testing Instructions for Keyboard


## Screenshots or screencast 

### Before

![Screenshot 2024-02-24 at 12 24 12](https://github.com/WordPress/gutenberg/assets/107534/c163bb7c-ce68-4ce5-9bfa-7dff759a5afb)



### After

![Screenshot 2024-02-24 at 12 22 50](https://github.com/WordPress/gutenberg/assets/107534/6de617ef-d693-4547-8375-ec2dc46067d7)
